tools/provision.py

```diff
@@ -80,6 +81,12 @@
     logging.critical("Only x86 is supported; ping zulip-devel@googlegroups.com if you want another architecture.")
     sys.exit(1)
 
+#Exit if RAM Size is less than 2GB
+ram_size = commands.getoutput("cat /proc/meminfo | head -1")
+if int(ram_size.split(' ')[-2])/(1024*1024) < 2:
```

Comment:
e.g. `lshw --class memory` or something similar to show installed memory, not just available.